tools/BaseRunner.py

```diff
@@ -1,3 +1,9 @@
+# Copyright (C) 2020  The SymbiFlow Authors.
```

Comment:
For non-runnable Python files I think we want the following (which makes sure that editors detect this file as Python 3?) -- Need to confirm this....
```
#!/bin/false python3
# -*- coding: utf-8 -*-
```